core/src/main/java/org/apache/iceberg/BaseTable.java

```diff
@@ -229,17 +229,17 @@ public Transaction newTransaction() {
 
   @Override
   public FileIO io() {
-    return operations().io();
+    return ops.io();
```

Comment:
These are unrelated , and just simplification/standardization, right?  Fine with me either way

Reply:
Where possible, I removed references to methods that expose `TableOperations` and deprecated the methods.

Here in `BaseTable`, I just wanted to be more consistent and use the instance field directly.